source/guide_roundcube.rst

```diff
@@ -0,0 +1,222 @@
+.. highlight:: console
+
+.. author:: Achim | pxlfrk <hallo@pxlfrk.de>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: mail
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/roundcube.svg
+      :align: center
+
+#################
+Roundcube Webmail
+#################
+
+.. tag_list::
+
+Roundcube webmail is a browser-based IMAP client with an easy-to-use user interface. It provides full functionality you expect from an email client, including MIME support, address book, folder manipulation, message searching and spell checking.
+
+The global Uberspace Webmail-Instance is also based on roundcube. Deploying your own roundcube webmail is ideal if you want to add functionality by plugins or customize the theme etc.
```

Comment:
It's not! :grin: we're using rainloop for uberspace 7.

Reply:
well ... my fault. I removed the paragraph.